src/src/DataStructs/WiFiEventData.h

```diff
@@ -122,7 +122,7 @@ struct WiFiEventData_t {
 
   unsigned long connectionFailures = 0;
 
-  std::map<int, uint32_t> connectDurations;
+  std::map<uint8_t, long> connectDurations;
```

Comment:
Doesn't this cause implicit casts to be applied? As the index is an `int` in most places where this is used, not an `uint8_t`.

Reply:
std::map is a template.
So each new combination of key and value type is another implementation of the same code.